src/linear_algebra/finite_dimensional.lean

```diff
@@ -71,6 +72,7 @@ end
 instance [finite_dimensional K V] (S : submodule K V) : finite_dimensional K S :=
 finite_dimensional_iff_dim_lt_omega.2 (lt_of_le_of_lt (dim_submodule_le _) (dim_lt_omega K V))
 
+/-- The dimension of a finite dimensional vector space, in ℕ-/
```

Comment:
```suggestion
/-- The dimension of a finite-dimensional vector space is in ℕ -/
```

Reply:
Not sure of this change: your phrasing seems to be about a Prop, which is not what the definition does. What about `/-- The dimension of a finite dimensional vector space as a natural number -/`?

Reply:
Yes, that works better.
```suggestion
/-- The dimension of a finite-dimensional vector space as a natural number -/
```